lib/stream_data.ex

```diff
@@ -701,20 +722,48 @@ defmodule StreamData do
   #       |> LazyTree.flatten()
   #     end)
   #
-  @spec list_of(t(a)) :: t([a]) when a: term
-  def list_of(data) do
+  @spec list_of(t(a), keyword) :: t([a]) when a: term
+  def list_of(data, options \\ []) do
+    list_length_range_fun = list_length_range_fun(Keyword.take(options, [:length, :min_length, :max_length]))
```

Comment:
Ah, I did this with the idea that I would reuse this function in `uniq_list_of` which takes more options. Probably not needed in that case anyways.